components/instructions/instructionPanel.tsx

```diff
@@ -14,6 +14,7 @@ import {
 
 export function InstructionPanel() {
   const { instructions, proposal } = useProposal()
+  console.log({ instructions })
```

Comment:
nit: remove console log

Reply:
Sorry for late reply, I removed the console.log